ccmlib/scylla_node.py

```diff
@@ -598,8 +585,7 @@ def start(self, join_ring=True, no_wait=False, verbose=False,
 
         # The '--kernel-page-cache' was introduced by
         # https://github.com/scylladb/scylla/commit/8785dd62cb740522d80eb12f8272081f85be9b7e from 4.5 version
-        current_node_version = self.node_install_dir_version
-        if current_node_version and LooseVersion('666.development') < LooseVersion(current_node_version) >= LooseVersion('4.5'):
+        if parse_version(self.cluster.version()) >= parse_version('4.5.dev'):
```

Comment:
`self.cluster.version()` function runs `nodetool`. But in this step the Scylla node is not started yet. So the nodetool can't run. This is the reason why property `node_install_dir_version` has been added.
How did you get it working?

Reply:
I believe that cluster.version() runs https://github.com/scylladb/scylla-ccm/blob/05dc1327966bffcc5bed144f1a7437c2ab85c862/ccmlib/cluster.py#L221-L222

If you refer to https://github.com/scylladb/scylla-ccm/blob/next/ccmlib/scylla_cluster.py#L266-L270, it's `ScyllaManager.version`, is that what you meant?

Reply:
@juliayakovlev ping

Reply:
Sorry for delay.
No, this is version of the node, not cluster:
https://github.com/scylladb/scylla-ccm/blob/f4f1f5c11b95c6eef3d4957e162953dc11956ba0/ccmlib/node.py#L1328

Reply:
@juliayakovlev but I changed the call to `self.cluster.version()`

Reply:
And which cluster version will be during upgrade cluster, when one node will run different version?

Reply:
Ok, fair enough, I pushed a new version that tries node_install_dir_version first and
falls back to cluster.version() only if the former could not find the version.